installing/disconnected_install/installing-mirroring-disconnected.adoc

```diff
@@ -16,35 +16,26 @@ include::modules/oc-mirror-about.adoc[leveloffset=+1]
-
-// About the mirror registry
-include::modules/installation-about-mirror-registry.adoc[leveloffset=+1]
-
-[role="_additional-resources"]
-.Additional resources
-
-* For information about viewing the CRI-O logs to view the image source, see xref:../../installing/validating-an-installation.adoc#viewing-the-image-pull-source_validating-an-installation[Viewing the image pull source].
-
 [id="prerequisites_installing-mirroring-disconnected"]
 == Prerequisites
 
-* You must have a container image registry that supports link:https://docs.docker.com/registry/spec/manifest-v2-2[Docker v2-2] in the location that will host the {product-title} cluster, such as Red Hat Quay.
+* You must have a container image registry, which is referred as the mirror registry, that supports link:https://docs.docker.com/registry/spec/manifest-v2-2[Docker v2-2] in the location that will host the {product-title} cluster, such as Red Hat Quay.
 +
 [NOTE]
 ====
 If you use Red Hat Quay, you must use version 3.6 or later with the oc-mirror plugin. If you have an entitlement to Red Hat Quay, see the documentation on deploying Red Hat Quay link:https://access.redhat.com/documentation/en-us/red_hat_quay/3/html/deploy_red_hat_quay_for_proof-of-concept_non-production_purposes/[for proof-of-concept purposes] or link:https://access.redhat.com/documentation/en-us/red_hat_quay/3/html/deploying_the_red_hat_quay_operator_on_openshift_container_platform/index[by using the Red Hat Quay Operator]. If you need additional assistance selecting and installing a registry, contact your sales representative or Red Hat Support.
 ====
 +
-If you do not already have an existing solution for a container image registry, subscribers of {product-title} are provided a xref:../../installing/disconnected_install/installing-mirroring-creating-registry.adoc#installing-mirroring-creating-registry[mirror registry for Red Hat OpenShift]. The _mirror registry for Red Hat OpenShift_ is included with your subscription and is a small-scale container registry that can be used to mirror the required container images of {product-title} in disconnected installations.
+If you do not already have an existing solution for a mirror registry, subscribers of {product-title} are provided a xref:../../installing/disconnected_install/installing-mirroring-creating-registry.adoc#installing-mirroring-creating-registry[mirror registry for Red Hat OpenShift]. 
+
+* The mirror registry must be reachable by every machine in the clusters that you provision. If the registry is unreachable, installation, updating, or normal operations such as workload relocation might fail. For this reason, you must run mirror registries in a highly available way, and the mirror registries must at least match the production availability of your OpenShift Container Platform clusters.
+
+* You have set the umask parameter to `0022` on the operating system that uses oc-mirror.
```

Comment:
should we say "you have to set the umask"

Reply:
Since it is prerequisites, and we would normally mention, what has been done. For example, "You have installed oc-mirror plugin." etc.